eval.go

```diff
@@ -75,23 +75,30 @@ func parseRemoteObject(v *runtime.RemoteObject, res interface{}) (err error) {
 		return
 	}
 
-	if v.Type == "undefined" || v.Value == nil {
+	value := v.Value
+	if v.Type == "undefined" || value == nil {
```

Comment:
Please change to:

```
if value == nil {
```

`value` should be nil when `v.Type == "undefined"`. If not, we should not try to hide that.

Reply:
yes